html/semantics/interactive-elements/the-dialog-element/non-modal-dialog-layout.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<link rel=author href="mailto:jarhar@chromium.org">
+<link rel=author href="mailto:falken@chromium.org">
+<link rel=help href="https://html.spec.whatwg.org/multipage/interactive-elements.html#the-dialog-element">
+<link rel=help href="https://bugs.chromium.org/p/chromium/issues/detail?id=382594">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<link href="resources/dialog-layout.css" rel="stylesheet">
```

Comment:
@josepharhar I think maybe this could be an inline `<style>` ?

Reply:
Done, it will sync to this PR shortly